build.rs

```diff
@@ -56,17 +60,53 @@ fn feature_check(filename: &str) -> bool {
     check_build.file(path).try_compile("check_perf_pt").is_ok()
 }
 
+fn build_libipt() {
+    eprintln!("Building libipt...");
+    env::set_current_dir(&Path::new(C_DEPS_PATH)).unwrap();
+    let res = Command::new("make")
+        .arg("libipt")
+        .output()
+        .unwrap_or_else(|_| panic!("Spawning the libipt build failed"));
```

Comment:
I'm not sure this error is accurate: other things than spawning could have failed? How about just "Fatal error when building libipt."?